tools/c7n_azure/tests/test_session.py

```diff
@@ -26,4 +27,5 @@ def test_api_version(self):
         s = Session()
         client = s.client('azure.mgmt.resource.ResourceManagementClient')
         resource = next(client.resources.list())
-        self.assertEqual('2018-04-01', s.resource_api_version(resource.id))
+        self.assertTrue(re.match('\\d{4}-\\d{2}-\\d{2}',
```

Comment:
Nice refactor so we don't break the build with this test when the version bumps up!

Reply:
Thanks :)